ducktape/tests/test.py

```diff
@@ -47,3 +52,55 @@ def run_standalone(cls):
 
         test.log_start()
         test.run()
+
+
+class TestLoader(Logger):
+
+    def is_test_file(self, file_name):
+        """test_*.py or *_test.py"""
+        pattern = "(^test_.*\.py$)|(^.*_test\.py$)"
+        return re.match(pattern, file_name) is not None
+
+    def discover(self, base_dir):
```

Comment:
A short doc about what is discovered (either here or at the class level) would be useful. Helps for anyone who is reading the code or might want to extend it a bit.